crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -673,6 +706,9 @@ fn encryption_state_default() -> bool {
 }
 
 impl RoomInfo {
+    /// The size of the latest_encrypted_events RingBuffer
+    const MAX_ENCRYPTED_EVENTS: usize = 10;
```

Comment:
One thing that worries me here is that this now inflates the size of the `RoomInfo`. And every `RoomInfo` for each room we know about gets deserialized at `Client` construction time.

That's something we need to fix anyways, but it was fast enough for now, while it might not be the case anymore. This is especially true for freshly logged in clients, which will have a lot of UTDs.

Reply:
Resolution will be to move the encrypted events into `Room`. Working on it.

Reply:
Done in e900d8eab475b00774bbdc80e683de7b1529a11f